react/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "@vtex/tsconfig",
   "compilerOptions": {
+    "jsx": "react",
```

Comment:
Is this necessary?

Reply:
@victorhmp I think this happened on the "vtex link" command 🤔 I didn't edit this file..

Reply:
😂  No problem